### DISCUSSION TITLE
Glow used in Yazi, no custom styles?

### DISCUSSION THREAD

**Discussion by resistorsoftware** [charmbracelet/glow#662] [Q&A]
I run glow from the command-line, and it picks up my custom stylesheet where I set headers to be what I like. So that is great. But when I fire up Yazi, and view a markdown file, it uses Glow to render but glow stubbornly renders with a default markdown stylesheet ignoring my custom one. How does one trick glow into rendering with the custom styles setup in the glow config file for an installation? If that config is over-ridden, how does one override the override?

**Comment by caarlos0** [comment]
you can run `glow config` and change the style there to the one you want

if that's what you did, not sure I follow how you're getting this issue... does yazi fires up `glow` automatically? how does it work?

**Reply by resistorsoftware** [reply]
I was finding glow at the command-line would render a markdown file differently than glow being used as a plugin to Yazi, where the  rendered markdown preview was not matching my expectations. 

I eventually fiddled enough to where I understood why, but I ran out the energy to do much about it. It is not a terribly serious issue, but instead, one of tweakers delight.

**Comment by gennaro-tedesco** [comment]
> you can run glow config and change the style there to the one you want

explicitly indicating a path to a custom json style as indicated [here](https://github.com/charmbracelet/glow?tab=readme-ov-file#the-config-file) seems not to work, though. I too have been trying to have glow custom styles work in yazi and it seems the underlying problem is that the yml reference to the json style doesn't work as intended.
